content/en/docs/ops/integrations/prometheus/index.md

```diff
@@ -146,7 +167,18 @@ tls_config:
   ca_file: /etc/prom-certs/root-cert.pem
   cert_file: /etc/prom-certs/cert-chain.pem
   key_file: /etc/prom-certs/key.pem
-  insecure_skip_verify: true  # Prometheus does not support Istio security naming, thus skip verifying target pod ceritifcate
+  insecure_skip_verify: true  # Prometheus does not support Istio security naming, thus skip verifying target pod certificate
```

Comment:
These paramters should be camelCase, not snake_case:
```
caFile: .. 
certFile: ..
keyFile: ..
insecureSkipVerify: .. 
```

Also it could be mentioned that PodMonitor does not support the file attributes at all :/